### ISSUE TITLE
Warning in header in MSVC

### ISSUE THREAD

**Issue by chfast** [CLIUtils/CLI11#157]
```
c:\.hunter\_base\15e27f2\4668f98\6a3ff3d\install\include\cli\configfwd.hpp(120): warning C4458: declaration of 'items' hides class member [C:\projects\evmc\build\test\vmtester\evmc-vmtester.vcxproj]
  c:\.hunter\_base\15e27f2\4668f98\6a3ff3d\install\include\cli\configfwd.hpp(63): note: see declaration of 'CLI::Config::items'
```

**Comment by henryiii** [comment]
The warning is correct, we need to rename the variable in that function.

**Comment by henryiii** [comment]
Please verify this is now fixed in master through #159.
